apps/checker/conf/application.conf

```diff
@@ -11,6 +11,7 @@ play.assets {
 typerighter {
   sheetRange="A:G"
   defaultAwsProfile = "composer"
+  checkerServiceUrl = "typerighter.local.dev-gutools.co.uk"
```

Comment:
I think the url should include the prefix here - **checker**.typerighter.local.dev-gutools.co.uk.

But I can't see where this config value is used, and I'm not sure if this is how we should be using config? 

This config describes the checker app's own features, rather than configuring the checker app to use a particular property. 

It makes sense when we use it in the manager's config, but I'm not sure it makes sense here.

Reply:
Good catch. This was likely left behind when I thought that the entire request URL was signed by HMAC. Turns out, it's just the URL's path, so this isn't necessary – removed in 0b64a29ecf9ce4c9189225214d5cf0bf02197c5c.